packages/client/src/components/CartSummary/SideCartItem.tsx

```diff
@@ -13,7 +15,15 @@ const SideCartItem = ({ dish, count, price }: DishProps) => {
   const { name, photo, fats, proteins, carbohydrates, calories } = dish;
   return (
     <Box mb={2} display='flex' alignItems='center'>
-      <Box component='img' mr={1} width='5.3125rem' height='5.3125rem' borderRadius='50%' src={photo} alt={name} />
+      <Box
```

Comment:
```suggestion
      <Box sx={{ objectFit: 'cover' }}
```
To jest poza scope Twojego taska, ale znalazłam tu rozjechane zdjęcie to można to od razu poprawić.